File: ChickensAndRabbits.py
heads = 35
legs = 94
chicken = heads - 1
rabbit = 1

print(f"Heads: {heads}, Legs: {legs}\n")

for _ in range(heads):
    if chicken * 2 + rabbit * 4 == legs:
	    print(f"Chickens: {chicken}\nRabbits: {rabbit}")
	    break
    chicken -= 1
    rabbit += 1

File: ChickensAndRabbits.cpp
#include <iostream>

int main()
{
    int heads = 35;
    int legs = 94;
    int chicken = heads - 1;
    int rabbit = 1;

    std::cout << "Heads\t: " << heads;
    std::cout << "\nLegs\t: " << legs;

    for (int x = 0; x < heads; x++)
    {
        if (2 * chicken + 4 * rabbit == legs)
        {
            std::cout << "\nChickens: " << chicken;
            std::cout << "\nRabbits\t: " << rabbit;
            break;
        }
        chicken--;
        rabbit++;
    }
}
